# Patch submitted to force ply support bundled with source.
# Hector Martin <hector@marcansoft.com>
# Further info can be found at http://bugs.gentoo.org/show_bug.cgi?id=276018

--- mozilla-1.9.1/xpcom/idl-parser/xpidl.py
+++ mozilla-1.9.1/xpcom/idl-parser/xpidl.py
@@ -42,7 +42,7 @@
 
 import sys, os.path, re
 
-sys.path.append(os.path.join(os.path.dirname(__file__), os.pardir, os.pardir,
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), os.pardir, os.pardir,
                              'other-licenses', 'ply'))
 from ply import lex, yacc
 

